seekr2/modules/common_cv.py

```diff
@@ -1966,7 +1966,7 @@ def find_voronoi_anchor_neighbors(anchors):
     
     try:
         triangulation = Delaunay(points)
-        vor = Voronoi(points)
+        voronoi = Voronoi(points)
```

Comment:
While this appears to be unused, it's a little weird that the tool suggests changes to variable names.

Reply:
It was me reacting on tool finding vor to be a close hit to a typo.  I did that in a separate commit. If you would like I can just remove variable altogether or a full line - wasn't sure if code doesn't rely on some side effects there